.github/workflows/build.yml

```diff
@@ -7,10 +7,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 1.8
+    - name: Set up JDK 11
       uses: actions/setup-java@v1
       with:
-        java-version: 1.8
+        java-version: 11
```

Comment:
Seeing as this pull request is still in draft status, can we assume that the Java version will be changed back to 8 before official review? Otherwise, I really think that this should be discussed *beforehand* (and in a designated issue, not buried here).

Reply:
You are right - not a good idea!
Downgraded to use jena to the last java 8 compatible version.